.travis.yml

```diff
@@ -21,6 +21,7 @@ after_success:
 - openssl aes-256-cbc -K $encrypted_ecdba13b2a41_key -iv $encrypted_ecdba13b2a41_iv -in deployment_keys.tar.enc -out deployment_keys.tar -d
```

Comment:
Please introduce different lines for the different tests (unit, components, ...) instead of just executing `npm run test`

Reply:
The "unit" folder contains:
- an empty test file
- a trial test file with some default values

In my opinion having a separate `npm:unit` command is currently irrelevant. Could be added later when "unit" test are written.

Reply:
We do not need to run unit-tests as of Travis but I think it's useful to have a unit test command in the package.json, even though there are no useful tests yet. Otherwise people might come up with a different technology or package for unit-tests than what we planned to use.